adoc/admin-security-rbac.adoc

```diff
@@ -134,10 +139,13 @@ Besides the LDAP connector you can also set up other connectors.
 For additional connectors, refer to the available connector configurations in the Dex repository:
 https://github.com/dexidp/dex/tree/v2.16.0/Documentation/connectors.
 +
-. Update Dex deployment:
+. Save and exit Dex ConfigMap by typing `:wq` in the terminal.
```

Comment:
This is only needed when using `vi/vim` to edit the file. We don't specify the use of this editor. Please rephrase to "Save the changed ConfigMap".

Reply:
I would rather let it be as it is because `kubectl edit configmap whatever` is opening the file in `vim` by default (or editor set in $EDITOR env. variable but by default it's `vim`). 

With the proposed change some non-unix users might be confused and will not know how to save&exit.

Reply:
I agree with @thehejik and would leave that part in. How likely is it that people will go change the $EDITOR env. variable on the virtual machine?

Reply:
OK agreed. If this opens vim automatically then this makes sense.